monitoring/replication/alerts.test.yaml

```diff
@@ -0,0 +1,455 @@
+        values: 1+1x3 5+11x4 60+101x6
+      - series: replication_status_changed_total{namespace="zenko", job="backbeat-replication-status-processor-headless", replicationStatus="COMPLETED"}
+        values: 9999+9999x15
+    alert_rule_test:
+      - alertname: ReplicationErrorsWarning
+        eval_time: 5m
+        exp_alerts: []
+      - alertname: ReplicationErrorsCritical
+        eval_time: 5m
+        exp_alerts: []
+
+      - alertname: ReplicationErrorsWarning
+        eval_time: 9m
+        exp_alerts: []
+      - alertname: ReplicationErrorsCritical
+        eval_time: 9m
+        exp_alerts: []
+
+      - alertname: ReplicationErrorsWarning
+        eval_time: 10m
+        exp_alerts:
+          - exp_annotations:
+              description: Replication error rate is higher than 0.1%
+              summary: High ratio of replication errors
+            exp_labels:
+              severity: warning
+      - alertname: ReplicationErrorsCritical
+        eval_time: 10m
+        exp_alerts: []
+
+      - alertname: ReplicationErrorsWarning
+        eval_time: 14m
+        exp_alerts:
+          - exp_annotations:
+              description: Replication error rate is higher than 0.1%
+              summary: High ratio of replication errors
+            exp_labels:
+              severity: warning
+      - alertname: ReplicationErrorsCritical
+        eval_time: 14m
+        exp_alerts: []
+
+      - alertname: ReplicationErrorsWarning
+        eval_time: 15m
+        exp_alerts:
+          - exp_annotations:
+              description: Replication error rate is higher than 0.1%
+              summary: High ratio of replication errors
+            exp_labels:
+              severity: warning
+      - alertname: ReplicationErrorsCritical
+        eval_time: 15m
+        exp_alerts:
+          - exp_annotations:
+              description: Replication error rate is higher than 1%
+              summary: Very high ratio of replication errors
+            exp_labels:
+              severity: critical
+
+  - name: Rpo
+    interval: 1m
+    input_series:
+      # Requests last 50s for 5 min, then up to 600s for 5 min, then 900s for 5min
+      - series: replication_rpo_seconds_sum{namespace="zenko", job="backbeat-replication-data-processor-headless"}
+        values: 500+500x4 3000+6000x4 33000+9000x6
+      - series: replication_rpo_seconds_count{namespace="zenko", job="backbeat-replication-data-processor-headless"}
+        values: 10+10x16
+    alert_rule_test:
+      - alertname: RpoWarning
+        eval_time: 6m
+        exp_alerts: []
+      - alertname: RpoCritical
+        eval_time: 6m
+        exp_alerts: []
+
+      - alertname: RpoWarning
+        eval_time: 11m
+        exp_alerts:
+          - exp_annotations:
+              description: |
+                RPO is higher than 600 seconds.
```

Comment:
Shouldn't we more specific and say `Replication RPO` ? Similar question for the alert name